pkg/services/sqlstore/user.go

```diff
@@ -399,6 +400,17 @@ func UpdateUser(ctx context.Context, cmd *models.UpdateUserCommand) error {
 	})
 }
 
+func UpdateUserProductTour(ctx context.Context, cmd *models.UpdateUserProductTourCommand) error {
+	return inTransaction(func(sess *DBSession) error {
+		user := models.User{
+			ProductTourDone: cmd.ProductTourDone,
```

Comment:
`Updated` field is missing

Reply:
Frankly, I would not mess with native grafana entities like `user` or anything similar. Think about the maintenance effort: grafana will mutate the structure and we won't know about it. On every upgrade it will turn to a nightmare. Ask @fabio-silva, he will share his prior experience.

Reply:
Nor would I recommend to touch the original code, all for the same token.

Reply:
Yep. The less we change their own code, the better. Unless strictly necessary, of course.